pnpm-workspace.yaml

```diff
@@ -202,8 +202,8 @@ catalogs:
   # @see: https://mui.com/system/styles/basics/
   # climatemappedafrica, pesayetu and promisetracker still use @mui/styles
   mui-styles:
-    "@mui/icons-material": ^5.16.6
+    "@mui/icons-material": ^6.4.0
     "@mui/lab": ^5.0.0-alpha.155
-    "@mui/material": ^5.16.6
-    "@mui/styles": ^5.16.6
-    "@mui/utils": ^5.16.6
+    "@mui/material": ^6.4.0
+    "@mui/styles": ^6.4.0
+    "@mui/utils": ^6.4.0
```

Comment:
As discussed in the call:

1. We shouldn't upgrade the `@mui` version(s) in the `mui-styles` catalogue.
2. `climatemappedafrica` shouldn't be included in "climatemappedafrica, pesayetu and promisetracker still use @mui/styles" anymore;